src/Illuminate/Database/Eloquent/JsonEncodingException.php

```diff
@@ -17,4 +17,17 @@ public static function forModel($model, $message)
     {
         return new static('Error encoding model ['.get_class($model).'] with ID ['.$model->getKey().'] to JSON: '.$message);
     }
+
+    /**
+     * Create a new JSON encoding exception for an attribute.
+     *
+     * @param  mixed  $key
+     * @param  string $message
+     * @return static
+     * @internal param mixed $key
```

Comment:
@lucasmichot Agreed. It slipped by me as I was renaming the parameter. I'll correct in a second.